docs/concepts/feature-generation.md

```diff
@@ -32,51 +29,3 @@ settings = MaterializationSettings("nycTaxiMaterializationJob",
 client.materialize_features(settings)
 ```
 Alternatively, this can be done by using feature generation config and CLI:
```

Comment:
Remove this "Alternatively, this can be done by using feature generation config and CLI:"